Source/Scene/Expression.js

```diff
@@ -62,7 +62,16 @@ define([
         asin : Math.asin,
         atan : Math.atan,
         radians : CesiumMath.toRadians,
-        degrees : CesiumMath.toDegrees
+        degrees : CesiumMath.toDegrees,
+        sign : Math.sign,
+        floor : Math.floor,
+        ceil : Math.ceil,
+        round : Math.round,
```

Comment:
I'm having a lot of trouble with this. Currently getting `Vertex shader failed to compile.  Compile log: ERROR: 0:8: '0.5' : syntax error` but it's saying the error is in files I'm pretty sure I've never modified. 

Not really sure if I'm approaching this in the right way, anyway - I added in the special case for `round` and had to remove it from the unary functions, and in doing so had to add in a case for it in parseCall like I had done before the data-driven approach was implemented.